### PR TITLE
Add display support for Vertamedia Adapter

### PR DESCRIPTION
**Pull request by Millerrok**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change

Add display support for VertamediaBidAdapter

- test parameters for validating bids
```
      // Video instream adUnit
      {
        code: 'div-test-div',
        sizes: [[640, 480]],
        mediaTypes: {
          video: {
            context: 'instream'
          }
        },
        bids: [{
          bidder: 'vertamedia',
          params: {
            aid: 332842
          }
        }]
      }

      // Video outstream adUnit
      {
        code: 'outstream-test-div',
        sizes: [[640, 480]],
        mediaTypes: {
          video: {
            context: 'outstream'
          }
        },
        bids: [{
          bidder: 'vertamedia',
          params: {
            aid: 332842
          }
        }]
      }

      // Banner adUnit
      {
        code: 'div-test-div',
        sizes: [[300, 250]],
        bids: [{
          bidder: 'vertamedia',
          params: {
            aid: 324758
          }
        }]
      }
    
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

### CODE REVIEW

File: modules/vertamediaDisBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const URL = '//hb2.vertamedia.com/auction/';
+const BIDDER_CODE = 'vertamediadis';
+
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function (bid) {
+    return Boolean(bid && bid.params && bid.params.aid);
```
Comment (mkendall07) [inline modules/vertamediaDisBidAdapter.js]: Boolean cast isn't necessary.

**Comment by mkendall07** [comment]
@Millerrok 
The test creative ID is returning a vastURL and not an ad creative. Can you check please?

Request:
```
http://hb2.vertamedia.com/auction/?domain=ap.localhost&callbackId=299cffa7d44a55&aid=332842&ad_type=display&h=250&w=300&
```
Response:
```
{"source":{"aid":332842,"pubId":18016},"bids": [{"adid":"3BFE0D19E8A510E0","cmpId":211608,"requestId":"299cffa7d44a55","width":300,"height":250,"cpm":2,"url":"3BFE0D19E8A510E0","vastUrl": "http://ads59.vertamedia.com/vast/?adid=3BFE0D19E8A510E0","cur": "USD"}
]}
```

File: modules/vertamediaDisBidAdapter.js (hunk 1 above)
Comment (Millerrok) [reply modules/vertamediaDisBidAdapter.js]: @mkendall07 thanks, updated

**Comment by Millerrok** [comment]
@jaiminpanchal27  thanx for review. I have updated the PR according to your comments. Please take a look.

Thanks in advance, 
Miller

**Comment by mkendall07** [comment]
@Millerrok 
did you fix the test creative?

**Comment by Millerrok** [comment]
@mkendall07 yes, by changing parameter aid to 324758
 http://hb2.vertamedia.com/auction/?domain=ap.localhost&callbackId=299cffa7d44a55&aid=324758&ad_type=display&h=250&w=300&

**Comment by mkendall07** [comment]
@Millerrok 
Sorry for the delay. Can you please combine the video and display adapter? You can support multiple times in the adapter. Take a look here for example: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusBidAdapter.js#L10

**Comment by Millerrok** [comment]
@mkendall07 thanx for review. I have updated the PR and description according to your comments. Please take a look.

**Comment by Millerrok** [comment]
@mkendall07
Hi! We have added support for multi mediatypes for creating our bids 
Please take a look.

**Comment by mkendall07** [comment]
@Millerrok 
can you please open a PR for your docs update? 
https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/vertamedia.md

Thanks

File: test/spec/modules/vertamediaBidAdapter_spec.js [hunk 2]
```diff
@@ -42,72 +107,115 @@ describe('vertamediaBidAdapter', () => {
...
-        currency: 'USD',
-        height: 480,
-        width: 640,
-        ttl: 3600,
-        cpm: 0.9
-      }];
-
-      expect(result).to.deep.equal(eq);
+      console.log(result);
+      console.log(eqResponse);
```
Comment (matthewlane) [inline test/spec/modules/vertamediaBidAdapter_spec.js]: Please drop the above two `console.log` statements

Comment (Millerrok) [reply test/spec/modules/vertamediaBidAdapter_spec.js]: @matthewlane 
Code has been updated
Please take a look.

**Comment by Millerrok** [comment]
@mkendall07 
Yes, we Update Vertamedia Adapter description, and add examples
https://github.com/prebid/prebid.github.io/pull/567

Please take a look.

**Pull request merged**